README.md

```diff
@@ -594,19 +594,14 @@ MLJTuning.supports_heuristic(::TuningStrategy, ::SpecialHeuristic) = true
 ```
 
 
-### Implementation example: Search through an explicit list
+### Sample implementations
 
-The most rudimentary tuning strategy just evaluates every model
-generated by some iterator, such iterators constituting the only kind
-of supported range. The models generated must all have a common type
-and, in the implementation below, the type information is conveyed by
-the specified prototype `model` (which is otherwise ignored).  The
-fallback implementations for `extras` and `report_history` suffice.
+A number of built-in tuning strategy implementations of the MLJTuning
+API can be found at [/src/strategies](/src/strategies). 
 
-Unless the iterator `range` is exhausted, the `models`
-method returns returns `n_remaining` models.
-
-The `Explicit` implentation code is [here](/src/strategies/explicit.jl).
+The simplest `Explicit` strategy is the simplest but is also an odd
```

Comment:
Why is it called `Explicit`? Because it assumes less than the other wrappers?

Reply:
Because one explicitly lists the models rather than constructing them by cloning a prototype and mutating hyper-parameters over some range object (which is what all the other strategies, such as `Grid` or `RandomSearch` do).

Not in love with the name, so if  you have a better suggestion...

I think this part of the API is not really official, so we could change the name without making a breaking release. It would break absolutely nothing in the MLJ ecosystem.

Reply:
> Not in love with the name, so if you have a better suggestion...

So, we have a bunch of models and tuning it without cloning or mutating. The only suggestions that I can come up with are `Unchanged`, `FromObject`? But, yeah, I'm starting to get it and I see why the name is suitable